tools/updatetxids.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/python3
```

Comment:
simple tool I used to update the txids in the python files, leaving it here for later use.